### DISCUSSION TITLE
Rename docker-compose.yml to compose.yaml

### DISCUSSION THREAD

**Discussion by voltuer** [supabase/supabase#34976] [Feature Requests]
# Bug report

- [x] I confirm this is a bug with Supabase, not with my own application. 
- [x] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

Best practices from software vendors should be followed.

Source: https://docs.docker.com/compose/intro/compose-application-model/

## Expected behavior

Compose file should be named `compose.yaml` 

## System information

- OS: Windows

**Comment by voltuer** [comment]
Just submitted a PR that fixes this: https://github.com/supabase/supabase/pull/31163

**Comment by Hallidayo** [comment]
Hi everyone, due to inactivity on this issue I've moved the issue over to discussions/enhancements.

**Comment by voltuer** [comment]
is this contribution going anywhere?
